tests/integration/fund0xV2Trading.test.js

```diff
@@ -127,37 +128,40 @@ describe('Fund takes an order', () => {
       await call(accounting, 'getFundHoldingsForAsset', [mln.options.address])
     );
 
+    const orderAddresses = [];
+    const orderValues = [];
+    const orderData = [];
+
+    orderAddresses[0] = signedOrder.makerAddress;
+    orderAddresses[1] = signedOrder.takerAddress;
+    orderAddresses[2] = signedOrder.feeRecipientAddress;
+    orderAddresses[3] = signedOrder.senderAddress;
+    orderValues[0] = signedOrder.makerAssetAmount;
+    orderValues[1] = signedOrder.takerAssetAmount;
+    orderValues[2] = signedOrder.makerFee;
+    orderValues[3] = signedOrder.takerFee;
+    orderValues[4] = signedOrder.expirationTimeSeconds;
+    orderValues[5] = signedOrder.salt;
+    orderValues[6] = fillQuantity;
+    orderData[0] =  signedOrder.makerAssetData;
+    orderData[1] = signedOrder.takerAssetData;
+
+    const hex = web3.eth.abi.encodeParameters(
+      ['address[4]', 'uint256[7]', 'bytes[2]', 'bytes'],
```

Comment:
it might make sense to maintain a js mapping (in constants.js maybe) from `Integration` to ``EncodingTypes`. Maybe we can also expose this as a required variable on each adapter (see PR-level comment), and we can get the signature to encode by dynamically as we need it.